### DISCUSSION TITLE
S3 Upload not working

### DISCUSSION THREAD

**Discussion by eellllaa** [Budibase/budibase#6020] [Help]
hi, I'm trying to set S3 File Upload component but it's not working.

--

I set up correctly S3 Data (see the attachment) where I'm able to query it.
<img width="1680" alt="Screenshot 2022-05-22 at 18 49 19" src="https://user-images.githubusercontent.com/106027110/169706597-0c0fba83-1f0b-40b9-bec7-41bf2fced1ba.png">

Then I added the component in a form
<img width="1680" alt="Screenshot 2022-05-22 at 18 41 31" src="https://user-images.githubusercontent.com/106027110/169706624-11f6fc2b-09ff-4135-a206-75077f3e7afd.png">

I've also added the actions to the upload button
<img width="1680" alt="Screenshot 2022-05-22 at 18 49 40" src="https://user-images.githubusercontent.com/106027110/169706662-948a170d-77b0-4d1d-8e6b-bbe7301fd667.png">
<img width="1680" alt="Screenshot 2022-05-22 at 18 49 48" src="https://user-images.githubusercontent.com/106027110/169706665-57af9c86-b97f-4c4a-bdb8-84e5b3ead20e.png">
<img width="1680" alt="Screenshot 2022-05-22 at 18 49 58" src="https://user-images.githubusercontent.com/106027110/169706670-274f5f7d-3c8b-4eb2-8690-ffa8059da9c9.png">

But, it's not working and still get "Failed to send request" message.
<img width="1680" alt="Screenshot 2022-05-22 at 18 42 15" src="https://user-images.githubusercontent.com/106027110/169706697-8b4c8604-f408-42e0-91a7-0e1d9ded2481.png">

@aptkingston @mjashanks @shogunpurple Any idea about what I'm doing wrong? 

Thanks in advance for help.

**Comment by melohagan** [comment]
Hi @eellllaa 

Could you please share any errors you see in the network tab (Fn+F12 on Chrome).

**Reply by eellllaa** [reply]
<img width="1678" alt="Screenshot 2022-05-23 at 13 13 29" src="https://user-images.githubusercontent.com/106027110/169807224-1c2915f2-9bbe-46ec-864a-ab7835581d8b.png">


Also, even if the app has been set as public, I need to login to visit the page
https://filmimpresa.budibase.app/app/urt4tri6

**Reply by melohagan** [reply]
Appears to not be an issue with the request. If you share the screenshot of the console tab, the error should be in there.

**Reply by eellllaa** [reply]
<img width="1680" alt="Screenshot 2022-05-23 at 15 25 39" src="https://user-images.githubusercontent.com/106027110/169829599-66a73856-72b7-4c9c-bddf-b4fdca306c53.png">

**Reply by melohagan** [reply]
I think this is related: https://github.com/Budibase/budibase/discussions/5608

Are you using the latest version of Budibase?

**Reply by eellllaa** [reply]
I'm using cloud v 1.0.167 hosted by Budibase

**Reply by melohagan** [reply]
The work mentioned in the discussion I linked still needs to be done - it'll most likely be resolved sometime in June.
